abr_control/arms/threejoint/config.py

```diff
@@ -53,14 +53,15 @@ def __init__(self, **kwargs):
         self.REST_ANGLES = np.array([np.pi/4.0, np.pi/4.0, np.pi/4.0],
                                     dtype='float32')
 
-        # create the inertia matrices for each link of the threelink
+        # create the inertia matrices for each link of the three joint
         # TODO: identify the actual values for these links
-        self._M_LINKS.append(np.diag([10.0, 10.0, 10.0,
-                             0.0, 0.0, 10.0]))  # link0
-        self._M_LINKS.append(np.diag([10.0, 10.0, 10.0,
-                             0.0, 0.0, 10.0]))  # link1
-        self._M_LINKS.append(np.diag([10.0, 10.0, 10.0,
-                             0.0, 0.0, 10.0]))  # link2
+        self._M_LINKS.append(sp.zeros(6, 6))  # link0
+        self._M_LINKS.append(sp.diag(1.98, 1.98, 1.98,
+                             0.0, 0.0, 10.0))  # link1
+        self._M_LINKS.append(sp.diag(1.32, 1.32, 1.32,
+                             0.0, 0.0, 10.0))  # link2
+        self._M_LINKS.append(sp.diag(0.8, 0.8, 0.8,
+                             0.0, 0.0, 10.0))  # link3
```

Comment:
Why are the two first inertia values at zero and not 10.0?

Reply:
because the arm is restricted to a horizontal plane there's only one principle moment of inertia, around the z axis.